CBLClient/Apps/CBLTestServer-iOS/Frameworks/CouchbaseLiteSwift.framework/Headers/CBLErrors.h

```diff
@@ -0,0 +1,101 @@
+//
```

Comment:
avoid checking in this file. remove this file from this commit

Reply:
removed